client/src/renderer/redux/actions/pageStyles.js

```diff
@@ -0,0 +1,9 @@
+/** @format */
+
+import { SET_PAGE_STYLES } from "./types";
+
+export const setPageStyles = (key, data) => ({
```

Comment:
Just for the _key/value_ metaphor.
```suggestion
export const setPageStyles = (key, value) => ({
```